GOVERNANCE.md

```diff
@@ -0,0 +1,32 @@
+# Governance
+
+## Project Maintainers
+
+[Project maintainers](CODEOWNERS) are responsible for all activities
+maintaining and updating the Notary project. Final decisions on the projects
+reside with the project maintainers.
+
+Maintainers MUST remain active. To remain active they MUST collaborate on the 
+project at least once in a 3 month period. Example collaborations include, attending
+the community meeting, creation or commenting on issues/PRs, PR reviews, etc. 
+If they are unresponsive for >3 months, they will be automatically moved to emeritus unless a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
+the other project maintainers agrees to extend the period to be greater than 3
+months.
+
+New maintainers can be added to the project by a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
+vote of the existing maintainers. A potential maintainer may be nominated by an
+existing maintainer. A vote is conducted in private between the current
```

Comment:
I don't love the idea of private votes. Any reason to not do it in public on the issue through LGTMs? OCI has been using that process: https://github.com/opencontainers/distribution-spec/pull/288

Reply:
SGTM.  I'll reword based on the process you shared.